packages/ZbayLite/src/renderer/containers/widgets/settings/SettingsModal.tsx

```diff
@@ -10,7 +10,7 @@ import { useModal } from '../../hooks'
 const useSettingsModalData = () => {
   const data = {
     modalTabToOpen: useSelector(appSelectors.currentModalTab),
-    user: 'settingsModalUser',
+    user: useSelector(identity.selectors.currentIdentity).zbayNickname,
```

Comment:
This throws error when you open the app for the first time (cause there is no identity yet)
`Cannot read property 'zbayNickname' of undefined`

Reply:
Huh, I think it's a bigger problem. All the modals exists before we
initialize community data so this error will return whenever we use some
'fragile' selectors inside them. The question is will that be common
scenario or is it just a single case?

Reply:
Also, you shouldn't be able to reach that point without community at all